code/modules/projectiles/projectile.dm

```diff
@@ -908,7 +908,10 @@ So if we are on the 32th absolute pixel coordinate we are on tile 1, but if we a
 			feedback_flags |= (BULLET_FEEDBACK_FIRE)
 
 	if(proj.ammo.flags_ammo_behavior & AMMO_SUNDERING)
-		adjust_sunder(proj.sundering)
+		if(get_sunder() > 0.5)
+			adjust_sunder(proj.sundering)
```

Comment:
This has some issues with hardcapping sunder, if you want to hardcap it you have to make sure you limit the sunder the shot can apply or the very last sundering impact pre 50% sunder will "overflow" and lead to more sunder than should be possible (assuming the 50% "hard" limit for base projectiles)

Reply:
I know. If thats a big issue, ill fix it